executor/executor_bsd.h

```diff
@@ -38,9 +38,10 @@ static long execute_syscall(const call_t* c, long a[kMaxArgs])
 	return __syscall(c->sys_nr, a[0], a[1], a[2], a[3], a[4], a[5], a[6], a[7], a[8]);
 }
 
-#if GOOS_freebsd || GOOS_openbsd
+#if GOOS_freebsd || GOOS_openbsd || GOOS_netbsd
 
 // KCOV support was added to FreeBSD in https://svnweb.freebsd.org/changeset/base/342962
+// KCOV support added to NetBSD in https://github.com/NetBSD/src/commit/080a26f01d79ded8a2154c128950d1d663c6ca88
```

Comment:
I would skip this line or replace it with: `KCOV support was added to NetBSD 9.0.`